docs/LEARNING.md

```diff
@@ -0,0 +1,7 @@
+## Learning Rust
+
+* [The Rust Programming Language](http://doc.rust-lang.org/stable/book/) is a great resource for getting started with Rust as well as diving deeper into specific features of Rust.
+* [Rust by Example](http://rustbyexample.com) shows you examples of the most common things you will be writing in Rust.
+* [Rustlings](https://github.com/carols10cents/rustlings) "Small exercises to get you used to reading and writing Rust code."
+* [#rust-beginners](https://client00.chat.mibbit.com/?server=irc.mozilla.org&channel=%23rust-beginners) "an IRC channel that loves answering questions at any depth"
```

Comment:
Any strong reason to explicitly specify `client00` here instead of using `https://chat.mibbit.com/` and letting Mibbit load-balance among its `clientNN` subdomains itself? I'm not sure, but in the absence of strong reason to do otherwise I would prefer to omit `client00`